curvefs/src/client/warmup/warmup_manager.cpp

```diff
@@ -624,14 +689,23 @@ void WarmupManagerS3Impl::ScanWarmupFilelist() {
     if (!warmupFilelistDeque_.empty()) {
         WarmupFilelist warmupFilelist = warmupFilelistDeque_.front();
         VLOG(9) << "warmup ino: " << warmupFilelist.GetKey()
-                << " len is: " << warmupFilelist.GetFileLen();
+                << " len is: " << warmupFilelist.GetFileLen()
+                << " mount point is: " << warmupFilelist.GetMountPoint()
+                << " fs root is: " << warmupFilelist.GetRoot();
 
         std::vector<std::string> warmuplist;
         GetWarmupList(warmupFilelist, &warmuplist);
         VLOG(9) << "warmup ino: " << warmupFilelist.GetKey()
                 << " warmup list is: "
                 << fmt::format("{}", fmt::join(warmuplist, ","));
         for (auto filePath : warmuplist) {
+            size_t found = filePath.find(warmupFilelist.GetMountPoint());
+
+            if (found != std::string::npos) {
+                filePath.replace(found, warmupFilelist.GetMountPoint().length(),
+                                 warmupFilelist.GetRoot());
+            }
+
```

Comment:
Is the logic here to replace the actual path with the path in curvefs? It can be made into a function (or lambda expression) independently, and then the ones that cannot be replaced can be eliminated. There is no need to add it to the preheating task.

Reply:
extracted